requirements.txt

```diff
@@ -1 +1 @@
-opencv-contrib-python~=4.6.0.66  # for usage with GStreamer, one has to compile OpenCV with its support turned on
\ No newline at end of file
+#opencv-contrib-python~=4.6.0.66  # for usage with GStreamer, one has to compile OpenCV with its support turned on
```

Comment:
The final solution here would be to upload whl for our version of OpenCV somewhere and put it there as dependency. In the meantime, I put there at least some OpenCV. No problem if you comment it out for testing/development, but in the repo it should probably remain there until we have that whl somewhere publicly.

Reply:
OK